CONTRIBUTING.md

```diff
@@ -1,36 +1,60 @@
 # Contribution Guidelines for iNaturalistReactNative
-Welcome! We're excited that you're interested in contributing to the iNaturalist React Native mobile application. Below you'll find some guidelines that will help you contribute to our project.
+Welcome! We're excited that you're interested in contributing to the iNaturalist React Native mobile application. Below you'll find some guidelines that will help you contribute to our project, whether your on staff or a volunteer.
 
-We welcome volunteer contributions! This app is still early in its development and a lot of things are in flux, but there's usually something to work on. Please keep the following in mind:
+Please keep the following in mind:
 
-1. Work on [existing issues](https://github.com/inaturalist/iNaturalistReactNative/issues) (though if you have ideas for improvement that aren't directly related to features, let us know)
-1. Leave a comment on the issue so we can assign you and other people know not to work on it
+1. Work on [existing issues](https://github.com/inaturalist/iNaturalistReactNative/issues) (feature requests should go through the [Forum](https://forum.inaturalist.org/c/feature-requests/16))
+1. Leave a comment on the issue you want to work on so we can assign you and other people know not to work on it
 1. Name your branch starting with the issue number and then something descriptive, e.g. `123-login-with-locale-crash`
 1. We try to review pull requests as soon as we can, but that might be up to a week or two
 
 ## Getting Started
-1. Fork the repository.
-2. Clone the forked repository to your local machine.
-3. Follow [the README](README.md) to setup your local repository.
-4. Create a new branch for the issue you're working on; the branch name should start with the issue number and by concise but descriptive, e.g. `123-login-with-locale-crash`.
-5. Make changes to the codebase.
-6. Push your changes to your forked repository.
+1. Fork the repository
+2. Clone the forked repository to your local machine
+3. Follow [the README](README.md) to setup your dev environment
+4. Create a new branch for the issue you're working on; the branch name should start with the issue number and be concise but descriptive, e.g. `123-login-with-locale-crash`
```

Comment:
this feels a little silly, but should we specify that developers should pull (or rebase) from main before creating a new branch? every once in a while, I review a PR that makes me think some of us are working with outdated code

Reply:
Not silly at all, but we do have something to that effect in the "Submitting Changes" section. I'll re-arrange thing so  "Getting Started" and "Submitting Changes" don't overlap.